extras/ping/ping.go

```diff
@@ -0,0 +1,646 @@
+// Package ping is a simple but powerful ICMP echo (ping) library.
+// This file is a modification of one of the elements in the go-ping library.
+// for the purposes of writing raw UDP packets over a VPN tunnel.
```

Comment:
For this to work, I think you need to have the commend followed by `package ping` w/o any empty line